rust/relay/src/main.rs

```diff
@@ -34,6 +34,9 @@ struct Args {
     /// If omitted, the relay server will start immediately, otherwise we first log on and wait for the `init` message.
     #[arg(long, env)]
     portal_ws_url: Option<Url>,
+    /// Token generated by the portal to authorize websocket connection
+    #[arg(long, env)]
+    portal_token: String,
```

Comment:
This will need to be optional too then.

Reply:
the portal always requires the token, even for local testing

Reply:
But the relay does not always require the portal! If the URL is not given, we start without the portal connection. This is used for the local smoke test for example.